django_extensions/management/commands/print_user_for_session.py

```diff
@@ -39,19 +39,18 @@ def handle(self, *args, **options):
 
         print('Session to Expire: %s' % session.get_expiry_date())
         print('Raw Data: %s' % data)
-
         uid = data.get('_auth_user_id', None)
+        backend_path = data.get('_auth_user_backend', None)
```

Comment:
Can you change this to `from django.contrib.auth import BACKEND_SESSION_KEY, SESSION_KEY` ?

(I suggest fixing `_auth_user_id` at the same time :-) )

We also need an `if backend_path is None:` in case for whatever reason `_auth_user_backend` does not exist in the session and give some feedback to the user.